src/applications/hca/config/form.js

```diff
@@ -508,7 +510,12 @@ const formConfig = {
           title: 'Service history',
           uiSchema: {
             'ui:title': 'Service history',
-            'ui:description': 'Check all that apply to you.',
+            'ui:description': !environment.isProduction()
```

Comment:
Do you need this flag? None of the other changes appear to be behind a flag.

Reply:
> Do you need this flag? None of the other changes appear to be behind a flag.

@erikphansen , should I remove all conditional flags like the one above for the MilitaryPrefillMessage instances?  I don‘t know when the relevant epic/phase this belongs to (HCA update “phase 2”?).

Reply:
Flags removed.